tests/test_pythia8.py

```diff
@@ -98,18 +100,25 @@ def test_mothers(event):
     assert sum(x[1] >= -1 and x[1] < len(event) for x in event.mothers) == n
 
 
-@pytest.mark.skip(reason="Simulating nuclei in Pythia8 is very time-consuming")
-def test_nuclear_collision():
-    # The test takes ages because the initialization is extremely long,
-    # and Pythia seldom raises the success flag unless Ecm > TeV are used.
-
-    event = run_collision(2000 * GeV, "p", (4, 2))
-    assert event.pid[0] == 2212
+@pytest.mark.parametrize("projectile", ("pi-", "K+", "p", "He"))
+def test_nuclear_collision(projectile):
+    # This test is very slow the first time when Pythia is computing
+    # some cached results. Afterwards it runs super fast.
+    ecm = 2000 * GeV
+    event = run_collision(ecm, projectile, "He")
```

Comment:
Here the cache should be enabled using files distributed via the binary package in iamdata.